src/app/room/room.cpp

```diff
@@ -58,3 +62,14 @@ void Room::createRoom(const unsigned room_id, const std::vector<Treatment>& trea
     Room room{room_id, treatments, warehouse};
     Clinic::appendRoom(std::make_shared<Room>(room));
 }
+
+void Room::createRoom(const unsigned room_id, const std::shared_ptr<Warehouse>& warehouse)
+{
+    Room room{room_id, warehouse};
+    Clinic::appendRoom(std::make_shared<Room>(room));
+}
```

Comment:
Just curious if it could be shortened to 
`Clinic::appendRoom(std::make_shared<Room>({room_id,warehouse}));`?

Reply:
Looks like it does, need to implement it in every create function